Rework the devtools theme customization page (#1342)
MicrosoftDocs/edge-developer: microsoft-edge/devtools-guide-chromium/accessibility/preferred-color-scheme-simulation.md

> ---

Reviewer: description: Force Microsoft Edge DevTools into Color Scheme Preview mode.
That description implies that the article is about changing the theme of the DevTools portion of the window:
Is the article actually about changing the rendered webpage portion of the window instead?

The tab text in the browser shows "Force Microsoft Edge DevTools", which reads oddly.  The tab is probably populated by the "title:" field.  A useful string for one of the YAML fields might be "Render the webpage with light or dark theme emulation".

Update the "date:".

Make the "title:" field match the # (top-of-page) title.  Currently those are:
title: Force Microsoft Edge DevTools into Color Scheme Preview mode (CSS Prefers Color Scheme)
# Dark or light color scheme simulation 
If you change the # title, check the TOC.yml too and do full-text search in repo for the present filename, to update link text in any articles that link to the present article.

Author: I would personally prefer to keep the title shorter and drop the "in the rendered page" part. My reasoning for this is that all other pages in the Rendering section are about changing things in the page (as opposed to devtools). So I don't think there is too much of a risk for readers to get confused as to where the color scheme will apply, especially if the content of the article makes it clear.

> ---

Reviewer: Update the "date:" field.

> ---

Reviewer: Clarify the "description:" field.
`description: Force Microsoft Edge DevTools into Color Scheme Preview mode.`
At a glance, that description seems to imply that the article is about changing the theme of the DevTools portion of the window.  Is the article actually about changing the rendered webpage portion of the window instead?  It would be good to draw that distinction more explicitly.

# Dark or light color scheme simulation  

> 

Reviewer: Needs a clarifying point explicitly up front: Is this article about changing the theme of the DevTools portion of the window, or changing the rendering of the webpage?  ( = the Appearance commands vs the Rendering commands in the Command Menu.)

> Many operating systems have a way to display any application in darker or lighter colors.  Having a web product that has a light theme in a dark-mode operating system can be hard to read and can be an accessibility issue for some users.  

Reviewer: Globally in article, change "theme" to "scheme".
Apparently the code and UI uses the word "scheme", not "theme", for this subject.

> For a webpage, you can use the [prefers-color-scheme][MDNPrefersColorScheme] CSS Media Query to detect whether the user prefers to display your product in a darker or lighter color scheme.  Then to test the result, use [Microsoft Edge DevTools][DevtoolsIndex] to simulate a change from dark to light mode, without having to change the setting for your entire operating system.  

Reviewer: It seems like this paragraph is actually an important required action step that belongs in the following numbered list.  The Accessibility Overview article or child articles shows specific example code to do this, together with this command/setting that's in the numbered list.  Not sure if the present link goes to that article.

**To emulate dark or light theme:**

1.  Open the **Command Menu** by selecting `Ctrl`+`Shift`+`P` \(Windows/Linux\) or `Command`+`Shift`+`P` \(macOS\).  

>     :::image type="complex" source="../media/css-console-command-menu-rendering.msft.png" alt-text="The Command Menu" lightbox="../media/css-console-command-menu-rendering.msft.png":::

Reviewer: Delete this extra, superfluous image.  It shows typing-in the word "Rendering", which is not recommented.  Fairly confusing.   https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/accessibility/preferred-color-scheme-simulation?branch=pr-en-us-1342
@Reezaali - pretty confusing; recommend fixing before merge this PR.

        The **Command Menu**  
    :::image-end:::  

> 1.  Type "emulate color", choose either **Emulate CSS prefers-color-scheme: dark** or **Emulate CSS prefers-color-scheme: light**, and then select **Enter**.  

Reviewer: ```suggestion
1.  Type "dark" or "light", select **Rendering: Emulate CSS prefers-color-scheme: dark** or **Rendering: Emulate CSS prefers-color-scheme: light**, and then select **Enter**.  
```
Simply typing "dark" or "light" is more realistic and advisable than the harder-to-type & harder-to-remember phrase, "emulate color".  But changing this text requires redoing the screen capture too.

>     :::image type="complex" source="../media/css-elements-styles-qs-select-renderingmode-command-menu.msft.png" alt-text="Color scheme option from Command Menu" lightbox="../media/css-elements-styles-qs-select-renderingmode-command-menu.msft.png":::

Reviewer: In screenshot, change the entered text to just "dark" or just "light".

       Color scheme option from **Command** Menu  
    :::image-end:::  
    
    > [!IMPORTANT]

>     > Simply typing `dark` or `light` does not reveal the right tool, since there is also a way to [choose a theme for DevTools][DevtoolsCustomizeTheme].  Select a **Rendering** command, rather than an **Appearance** command.  

Reviewer: ```suggestion
    > Select a **Rendering** command rather than an **Appearance** command.  The **Rendering** commands affect the rendered webpage.  The **Appearance** commands instead affect the DevTools part of the window, as described in [Apply color themes to Microsoft Edge DevTools][DevtoolsCustomizeTheme].
```

>     :::image-end:::  

Reviewer: At top of article, change:
Having a web product that has a light theme in a dark-mode operating system is grating
to:
Having a web product that has a light theme in a dark-mode operating system can be hard to read

    
    > [!IMPORTANT]

>     > Simply typing `dark` or `light` does not reveal the right tool, since there is also a way to [choose a theme for DevTools][DevtoolsCustomizeTheme].  If you are wondering what to choose, make sure that you are choosing a **rendering** menu item, not an **appearance** menu item.  

Reviewer: ```suggestion
    > Simply typing `dark` or `light` does not reveal the right tool, since there is also a way to [choose a theme for DevTools][DevtoolsCustomizeTheme].  Select a **Rendering** command, rather than an **Appearance** command.
```

> 

Reviewer: Shrink the .png below "Type emulate color, choose either Emulate CSS prefers-color-scheme: dark or Emulate CSS prefers-color-scheme: light and then select Enter."   eg:
1.  In VS Code, in this .md file, Ctrl+click the image path.  
2. Shift+Alt+R (or some menu item) to open Explorer with the image dir.  
3. Right-click the image, select Edit, opens in Paint.  
4. Select Resize, %, often 50% works, this might need 60%.  Save in Paint.
Unconfirmed whether ::: image syntax has % size options.

> 

Reviewer: minor suggestion for slight clarif; consistently separated actions by commas:
Change:
Type emulate color, choose either Emulate CSS prefers-color-scheme: dark or Emulate CSS prefers-color-scheme: light and then select Enter.
to:
Type `emulate color`, select either **Emulate CSS prefers-color-scheme: dark** or **Emulate CSS prefers-color-scheme: light**, and then select `Enter`.

Markup notes:
(some other doc sets would use **Enter**)
(some other doc sets would use "emulate color")

> 

Reviewer: At bottom, change:
View and change your CSS like any other web page
to:
Modify your CSS and view the rendered result the same way as for any other web page.
